jade/cli/submit_jobs.py

```diff
@@ -29,7 +29,8 @@
     "-b", "--per-node-batch-size",
     default=DEFAULTS["per_node_batch_size"],
     show_default=True,
-    help="Number of jobs to run on one node in one batch."
+    help="Number of jobs to run on one node in one batch. If 0, auto-determine batches based on "
```

Comment:
This was the simplest implementation but is not necessarily the most intuitive for users. I'm open to suggestions.